fetch-content.sh

```diff
@@ -3,10 +3,18 @@
 # Die on any error for Travis CI to automatically retry:
 set -e
 
-if [ ! -e ~/.openra/Content/ra2/ra2.mix ]; then
+if [ "$(uname)" == "Darwin" ]; then
+	DIR="$HOME/Library/Application Support/OpenRA/Content/ra2/"
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
```

Comment:
maybe it would be simpler to put `else` here, because if it's not on Mac OS X then it's on Linux.